<launch>
    <arg name="map" default="map1_small"/>
    <arg name="gui" default="true"/>

    <include file="$(find gazebo_ros)/launch/empty_world.launch">
        <arg name="world_name" value="$(find simulation)/worlds/$(arg map).world"/>
        <arg name="paused" value="false"/>
        <arg name="use_sim_time" value="true"/>
        <arg name="gui" value="$(arg gui)"/>
        <arg name="headless" value="false"/>
        <arg name="debug" value="false"/>
    </include>

    <group ns="gazebo">
        <node pkg="simulation" type="gz_realtime_factor_node" name="gz_realtime_factor_node" output="screen"/>
    </group>

    <include file="$(find simulation)/launch/spawn.launch">
        <arg name="id" value="0"/>
        <arg name="x_pos" value="-2.0"/>
        <arg name="y_pos" value="0.0"/>
        <arg name="yaw" value="0.0"/>
    </include>

    <include file="$(find simulation)/launch/spawn.launch">
        <arg name="id" value="1"/>
        <arg name="x_pos" value="2.0"/>
        <arg name="y_pos" value="0.0"/>
        <arg name="yaw" value="0.0"/>
    </include>

</launch>
